doc/source/wellington_govt_nz.md

```diff
@@ -0,0 +1,50 @@
+# Wellington City Council
```

Comment:
There are some typos in the description. Please fix them.

Reply:
fixed